### PR TITLE
Dev stack bootstrap deploy

### PR DESCRIPTION
**Pull request by BJordi**
## Summary

- WBooks code was adapted to the frontend bootstrap structure.
- Services were set apart from controllers

## Trello Card

https://trello.com/c/XpoJnoTp/23-dev-stack-wolox-frontend-bootstrap

### CODE REVIEW

File: angular/src/app/library-config.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+angular.module('libraryModule').config([
```
Comment (SKOLZ) [inline angular/src/app/library-config.js]: use the `app.routes.js` file for this inside the app folder

File: angular/src/app/components/centered/dashboard/styles.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@import "https://fonts.googleapis.com/css?family=Robot";
+@import 'https://fonts.googleapis.com/css?family=Roboto';
```
Comment (SKOLZ) [inline angular/src/app/components/centered/dashboard/styles.scss]: if this file only contains styles related to the dashboard section change it's name to `dashboard.scss` move it to `app/src/scss/...` if it has more general styles.

File: angular/src/assets/vendor/css/olala.css [hunk 3]
```diff
@@ -0,0 +1 @@
+a { width: 12%; }
```
Comment (SKOLZ) [inline angular/src/assets/vendor/css/olala.css]: remove this

File: angular/src/scss/application.scss [hunk 4]
```diff
@@ -1,3 +1,3 @@
 @import 'variables/colors';
-@import 'styles';
-@import 'book_detail_styles';
+@import '../app/components/centered/dashboard/styles';
+@import '../app/components/centered/bookDetail/book_detail_styles';
```
Comment (SKOLZ) [inline angular/src/scss/application.scss]: camel case for file names

File: angular/src/app/components/centered/dashboard/styles.scss (hunk 2 above)
Comment (BJordi) [inline angular/src/app/components/centered/dashboard/styles.scss]: There is only one class that is related to many sections and one class that is related to dashboard from which many other classes from different sections extend. Should I put them aside in a general styles file? The other ones are exclusive from dashboard.

Comment (SKOLZ) [inline angular/src/app/components/centered/dashboard/styles.scss]: yes, you placed this file under `angular/src/app/components/centered/dashboard/` and the scss files placed there should reference styles applied to the dashboard view. Generic styles should be placed under the `app/src/scss/` path in a file named `commons.scss` or base.scss` or something like that

**Pull request merged**